doc/tutorial/statistical_inference/model_selection.rst

```diff
@@ -41,59 +41,67 @@ data in *folds* that we use for training and testing::
     >>> print(scores)
     [0.93489148580968284, 0.95659432387312182, 0.93989983305509184]
 
-.. currentmodule:: sklearn.cross_validation
+.. currentmodule:: sklearn.model_selection
 
 This is called a :class:`KFold` cross validation
 
 .. _cv_generators_tut:
 
-Cross-validation generators
-=============================
+Cross-validation classes
+========================
 
 
 
-The code above to split data in train and test sets is tedious to write.
-Scikit-learn exposes cross-validation generators to generate list
-of indices for this purpose::
+The above code, to split data into train and test sets, is tedious to write. 
+Scikit-learn provides a bunch of classes which can be used to generate lists
+of train/test indices based on popular cross-validation strategies.
 
-    >>> from sklearn import cross_validation
-    >>> k_fold = cross_validation.KFold(n=6, n_folds=3)
-    >>> for train_indices, test_indices in k_fold:
+These classes have a split method which generates the train/test indices.
+In the following example, let us use dummy values for X, to get the
```

Comment:
"let us use dummy values for X" -> "we generate train/test indices into an arbitrary array X, based on a 3-fold cross-validation strategy" ?  Trying to emphasize what's important vs what's simply incidental.